src/filters/Blur.ts

```diff
@@ -8,6 +8,12 @@ import type {
 } from './typedefs';
 import { isWebGLPipelineState } from './typedefs';
 import { classRegistry } from '../ClassRegistry';
+import { fragmentSource } from './blur.shaders';
+
+export const blurDefaultValues: Partial<TClassProperties<Blur>> = {
+  blur: 0,
+  mainParameter: 'blur',
```

Comment:
shouldn't this be static as well?

Reply:
(i already tried 3 times to remove it entirely or reduce its appareances)